var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -21,6 +21,9 @@ class PyMaestrowf(PythonPackage):
     version('develop', branch='develop')
     version('master',  branch='master')
 
+    # Development versions
+    version('1.1.9', tag='1.1.9dev1')
```

Comment:
```suggestion
    version('1.1.9dev1', tag='1.1.9dev1')
```
We want it to be clear that this isn't a stable release